apps/documents/views.py

```diff
@@ -0,0 +1,231 @@
+import json
+from functools import cache
+
+from django.contrib.auth.decorators import permission_required
+from django.contrib.auth.mixins import PermissionRequiredMixin
+from django.contrib.postgres.search import TrigramSimilarity
+from django.db import transaction
+from django.db.models import Q
+from django.shortcuts import get_object_or_404, redirect
+from django.urls import reverse
+from django.views.decorators.http import require_POST
+from django.views.generic import ListView, TemplateView
+
+from apps.documents.models import Repository, RepositoryType
+from apps.files.models import File
+from apps.teams.decorators import login_and_team_required
+from apps.teams.mixins import LoginAndTeamRequiredMixin
+
+
+class RepositoryHome(LoginAndTeamRequiredMixin, TemplateView):
+    template_name = "documents/repositories.html"
+
+    def get_context_data(self, team_slug: str, tab_name: str, **kwargs):
+        context = {
+            "active_tab": "manage_files",
+            "title": "Manage Files",
+            "tab_name": tab_name,
+            "files_list_url": reverse("documents:files_list", kwargs={"team_slug": team_slug}),
+            "upload_files_url": reverse("documents:upload_files", kwargs={"team_slug": team_slug}),
+            "collections_list_url": reverse("documents:collections_list", kwargs={"team_slug": team_slug}),
+            "new_collection_url": reverse("documents:new_collection", kwargs={"team_slug": team_slug}),
+            "files_count": File.objects.filter(team__slug=team_slug).count(),
```

Comment:
Should this filter files? Do we want to show assistant files, generated files etc?

Maybe we need to add a 'purpose' field? For now we could filter anything that has and 'external_id' since all assistant and generated files should have that.

Reply:
I'm not convinced that we should show assistant files here?

> Maybe we need to add a 'purpose' field?

I like this idea. I'll make a ticket to do so

Reply:
Exactly - I don't think we should but currently this isn't doing any filtering so we are right?

Reply:
Yeah you are correct. It didn't do any filtering, but we are filtering now based on `external_id`. I do like your suggestion of having a purpose field